SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -232,7 +232,7 @@ namespace ESpatialOSNetFlow
 {
 	enum Type
 	{
-		UnrealNativeNetworking,
+		NoAutomaticConnection,
 		SpatialOSLocalNetworking,
```

Comment:
let's rename the LocalNetworking and CloudNetworking to LocalDeployment and CloudDeployment to keep it consistent throughout the code. mentioning it here, but I also saw it being used in some methods in the SpatialGDKEditorToolbar.cpp